templates/telemetry/clients_daily_histogram_aggregates_v1.sql.py

```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/env python3
+"""clients_daily_histogram_aggregates query generator."""
+import sys
+import json
+import argparse
+import textwrap
+import subprocess
+import urllib.request
+
+
+PROBE_INFO_SERVICE = (
+    "https://probeinfo.telemetry.mozilla.org/firefox/all/main/all_probes"
+)
+
+p = argparse.ArgumentParser()
+p.add_argument(
+    "--agg-type",
+    type=str,
+    help="One of scalars/histograms/booleans/events",
+    required=True,
+)
+
+
+def generate_sql(opts, additional_queries, windowed_clause, select_clause):
+    """Create a SQL query for the clients_daily_histogram_aggregates dataset."""
+    get_keyval_pairs = """
+        CREATE TEMPORARY FUNCTION get_keyval_pairs(y STRING)
+        RETURNS ARRAY<STRING>
+        LANGUAGE js AS
+        '''
+          var z = new Array();
+          node = JSON.parse(y);
+          Object.keys(node).map(function(key) {
+            value = node[key].toString();
+            z.push(key + ":" + value);
+          });
```

Comment:
Since you want both the key and value, this could be written as:
```
for (let [key, value] of Object.entries(node)) {
  z.push(`${key}:${value}`);
}
```

I couldn't find any docs on what JS spec these UDFs support so maybe they don't like the templated strings (from ES2015)?